.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         uses: carlosperate/arm-none-eabi-gcc-action@48db4484a55750df7a0ccca63347fcdea6534d78
         with:
           release: ${{ matrix.gcc }}
-      - uses: seanmiddleditch/gha-setup-ninja@16b940825621068d98711680b6c3ff92201f8fc0
+      - run: sudo apt-get update && sudo apt-get install ninja-build
```

Comment:
This will always get the latest version, right? Can we expect here that the latest version is always compatible with our environment?

Reply:
The ninja package is stable enough to trust that it will. Although we can eventually switch to a devcontainer for all our CI jobs as well. This would pin all dependencies.